react/react1/week2/to-do-list/src/components/to-do-list.jsx

```diff
@@ -0,0 +1,117 @@
+import ListHeader from "./to-do-header";
+import "./css/task-item.css";
+import { useState } from "react";
+
+const myList = [
+  {
+    id: 1,
+    task: "Get out of bed",
+    deadLine: "Wed Sep 13 2017",
+    isCompleted: false,
+  },
+  {
+    id: 2,
+    task: "Brush teeth",
+    deadLine: "Thu Sep 14 2017",
+    isCompleted: true,
+  },
+  {
+    id: 3,
+    task: "Eat breakfast",
+    deadLine: "Fri Sep 15 2017",
+    isCompleted: false,
+  },
+];
+function ToDoList() {
+  const [tasks, setTasks] = useState(myList);
+
+  function AddTaskClickHandler() {
+    console.log("Add task clicked");
+    let maxId = 0;
+    for (let i = 0; i < tasks.length; i++) {
```

Comment:
this logic may disturb you in the future if you delete something from the middle of the array, for example